control-userlayer/src/main/java/org/oskari/control/userlayer/GetUserLayerStyleHandler.java

```diff
@@ -0,0 +1,51 @@
+package org.oskari.control.userlayer.data;
+
+import org.json.JSONObject;
+
+import fi.nls.oskari.annotation.OskariActionRoute;
+import fi.nls.oskari.control.ActionDeniedException;
+import fi.nls.oskari.control.ActionException;
+import fi.nls.oskari.control.ActionHandler;
+import fi.nls.oskari.control.ActionParameters;
+import fi.nls.oskari.control.ActionParamsException;
+import fi.nls.oskari.domain.map.userlayer.UserLayerStyle;
+import fi.nls.oskari.service.OskariComponentManager;
+import fi.nls.oskari.util.ConversionHelper;
+import fi.nls.oskari.util.ResponseHelper;
+import org.oskari.map.userlayer.service.UserLayerDbService;
+
+
+/**
+ * Expects to get layer id as http parameter "id".
+ */
+@OskariActionRoute("GetUserLayerStyle")
+public class GetUserLayerStyleHandler extends ActionHandler {
+    private final static String PARAM_ID = "id";
+    private UserLayerDbService userLayerDbService = null;
+    
+    @Override
+    public void init() {
+        super.init();
```

Comment:
Calling `super.init();` is unnecessary, since `ActionHandlers` implementation is empty.